consonance-arch/src/main/java/io/consonance/arch/containerProvisioner/ContainerProvisionerThreads.java

```diff
@@ -340,14 +359,67 @@ public Void call() throws Exception {
                 // TODO: need threads that each read from orders and another that reads results
                 do {
 
-                    LOG.debug("CHECKING FOR VMs TO REAP!");
+                    LOG.info("CHECKING FOR VMs TO REAP!");
+
+                    LOG.info("CHECKING DB FOR LOST JOB VMS TO REAP");
+
+                    // TODO: this could be dangerous if we loose a job but the worker is in endless mode and has picked up another since the machine previously
```

Comment:
Goes back to the comment in the readme, is this functional or should it be removed/kept?